server/services/gameLogic.js

```diff
@@ -72,32 +72,37 @@ exports.quitGame = player => {
         players = players.filter((otherPlayer) => otherPlayer !== player);
         socket.emitPlayers(getPlayers());
         return true;
-    } else {
-        // Game has started, player can't quit
-        return false;
     }
+    // Game has started, player can't quit, server is responsible for generating the error
+    return false;
 }
 
 /* Start the game with the players that have joined */
 exports.startGame = () => {
     if (status === statusTypes.NOT_STARTED && players.length >= minPlayers) {
         status = statusTypes.STARTED;
         nextRound();
-    } else {
-        // TODO: There aren't yet enough players in the game
-    }
+        return true;
+    } 
+    // There aren't yet enough players in the game, server is responsible for generating the error
+    return false;
 }
 
 /* End the game */
 exports.endGame = () => {
-    socket.emitEndGame();
-    status = statusTypes.NOT_STARTED;
-    currentPlayer = null;
-    currentWord = '';
-    roundNum = 0;
-    players = [];
-    playedCards = [];
-    votes = [];
+    if (status === statusTypes.GAME_OVER) {
+        socket.emitEndGame();
+        status = statusTypes.NOT_STARTED;
+        currentPlayer = null;
+        currentWord = '';
+        roundNum = 0;
+        players = [];
+        playedCards = [];
```

Comment:
could possibly group a couple of these up e.g.

cleanPlayerData() {
currentWord = '';
    players = [];	        roundNum = 0;
    playedCards = [];	        players = [];
    votes = [];	        playedCards = [];
}

and reuse the logic in the nextRound() method etc.